tf/utils.go

```diff
@@ -147,6 +147,12 @@ func NewPassReport(plan *tfjson.Plan) types.PassReport {
 }
 
 func NewCoverageReportFromState(state *tfjson.State) (coverage.CoverageReport, error) {
+	defer func() {
+		if r := recover(); r != nil {
```

Comment:
Is it possible to fix the panic instead of calling the `cover()`?

Reply:
This is to prevent the main process (which validates and generates the pass report) from crashing if the coverage report fails to generate. The coverage report might fail to generate due to the following reasons:

- The index file cannot be downloaded.
- The online swagger file cannot be downloaded.
- The swagger ref or variant cannot be correctly processed.